### PR TITLE
1384093 - RFE: API call to fail scheduled action for the specific systems

### PR DESCRIPTION
**Pull request by ogajduse**
According to BZ1384093 I did some changes. 

Here is the result after calling both variants of this call.
failedsystemspage

### CODE REVIEW

**Comment by jiridostal** [comment]
Wouldn't it be possible to use method `setStatus(ActionStatus statusIn)` from package `action.server.ServerAction`? Then we might not need extra SQL query.

File: java/code/src/com/redhat/rhn/common/db/datasource/xml/System_queries.xml [hunk 1]
```diff
@@ -2299,6 +2299,14 @@ ORDER BY E.update_date, E.id
   </query>
 </mode>
 
+<callable-mode name="fail_action_for_system">
```
Comment (ogajduse) [inline java/code/src/com/redhat/rhn/common/db/datasource/xml/System_queries.xml]: True, write-mode is more suitable for this, but no longer needed. I have decided to do it another way. See cb43a5584a766504e5dbabb6abe056b32652d38b

**Comment by ogajduse** [comment]
@jiridostal cb43a55 works fine. Can it be considered as the final solution or we should deal with deleting systemactions too?

**Comment by ogajduse** [comment]
thanks to all, implemented in f952fd3adde83cd958b4bff5571228aec89130cc

**Pull request closed without merging**